src/modules/bricks/host-bricks/host-bricks.html

```diff
@@ -74,7 +74,7 @@ <h5 ng-if="vm.brickList.length !== 0">{{vm.filteredbrickList.length}}
                     {{brick.port}}
                 </div>
                 <div class="ft-column ft-actions">
-                    <button class="btn btn-default launch-btn" ng-click="vm.redirectToGrafana(brick, $event)" uib-tooltip="Launch Dashboard" tooltip-placement="auto bottom-left">Launch Dashboard</button>
+                    <button class="btn btn-default launch-btn" ng-click="vm.redirectToGrafana(brick, $event)" tooltip-placement="auto bottom-left">Launch</button>
```

Comment:
@cloudbehl, according to https://github.com/Tendrl/ui/issues/651, 'Launch Dashboard' -> 'Dashboard'